python/beeai_framework/tools/search/wikipedia.py

```diff
@@ -25,9 +25,14 @@
 
 class WikipediaToolInput(BaseModel):
     query: str = Field(description="Search query, name of the Wikipedia page.")
-    full_text: bool = Field(description="If set to true will return the full text of the page.", default=False)
-    section_titles: bool = Field(description="If set to true returns section titles as the description.", default=False)
-    language: str | None = Field(description="Retrieves specified language version if available.", default=None)
+    full_text: bool = Field(
+        description="If set to true, it will return the full text of the page. Otherwise only the summary will be returned.",  # noqa: E501
```

Comment:
something like
```suggestion
        description=(
            "If set to true, it will return the full text of the page. "
            "Otherwise only the summary will be returned."
        ),  
```
should work to not add a noqa